### PR TITLE
Add standalone distribution

### PR DESCRIPTION
**Pull request by kenwheeler**
Adds webpack conf and npm task to build standalone distribution for use
in reduced test cases and demo code

cc @alexlande

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  cache: true,
+  entry: "./modules/index.js",
+  output: {
+    path: path.join(__dirname, '/dist'),
+    filename: 'Radium.js',
```
Comment (alexlande) [inline webpack.config.js]: Nit: can we lowercase the file name?

**Comment by alexlande** [comment]
Thanks for this :ship:

**Comment by alexlande** [comment]
Actually-- should we not bundle `React` in `dist`?

**Comment by kenwheeler** [comment]
Yeah, great call. What do you think about lodash?

File: webpack.config.js (hunk 1 above)
Comment (kenwheeler) [inline webpack.config.js]: Absolutely.

Comment (kenwheeler) [inline webpack.config.js]: Should the global var be lowercase too? 

```js
radium.MatchMediaItem
// vs
Radium.MatchMediaItem
```

Comment (alexlande) [inline webpack.config.js]: We've been using uppercase everywhere, but yeah, it probably should be lowercase.

**Comment by alexlande** [comment]
I think we can leave lodash in, it's just a few methods.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "Alex Lande"
   ],
   "scripts": {
+    "dist": "webpack --config ./webpack.config.js",
```
Comment (ryan-roemer) [inline package.json]: GLOBAL: Never prefix with `./`. Windows is compatible as long as you don't do that. (Internal `/` are fine).

Comment (ryan-roemer) [inline package.json]: NIT: You can also shorten this to `webpack` ;)

Comment (alexlande) [inline package.json]: Good call :) Thanks @ryan-roemer!

File: webpack.config.js (hunk 1 above)
Comment (ryan-roemer) [inline webpack.config.js]: I vote lower case file name, uppercase class name. See, e.g.,

* https://code.jquery.com/jquery-2.1.3.min.js
* https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.1.2/backbone-min.js
* https://cdnjs.cloudflare.com/ajax/libs/react/0.12.2/react.min.js

Comment (alexlande) [inline webpack.config.js]: Works for me

**Comment by ryan-roemer** [comment]
Also on the lower casing thing, be aware that in this specific PR from `dist/Radium.js` to `dist/radium.js` is likely going to mess things up on your mac's (which have a case insensitive file system even though it seems case sensitive). I'd recommend nuking all of `dist` first...

**Comment by kenwheeler** [comment]
Ok, updated everything per discussion

**Comment by alexlande** [comment]
Nice! :shipit:

**Comment by kenwheeler** [comment]
For the record, that mouse means merge?

**Comment by alexlande** [comment]
Yep :D
